src/app/dashboard-widgets/create-work-item-widget/create-work-item-widget.component.ts

```diff
@@ -44,8 +44,13 @@ export class CreateWorkItemWidgetComponent implements OnInit {
         active: true
       }])
       .switchMap(filters => this.workItemService
-        .getWorkItems(100000, filters))
+        .getWorkItems(100, filters)
```

Comment:
does this change matter for this fix ? backend will ignore above 100 anyway. 
I agree it should be fixed but lets keep changes to bare minimum to not avoid suprirsing sideeffects.